eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -327,8 +328,10 @@ protected final ForeignTojos scopedTojos() {
      * @throws TimeoutException If timeout limit reached
      */
     private void execWithTimeout() throws ExecutionException, TimeoutException {
+        ExecutorService service = null;
```

Comment:
@c71n93 Maybe we can initialise `service` above the `try-catch` statement? In this case we won't need to check `null`:
```java
        final ExecutorService service = Executors.newSingleThreadExecutor();
        try {
            service.submit(
                () -> {
                    this.exec();
                    return new Object();
                }
            ).get(this.timeout.longValue(), TimeUnit.SECONDS);
        } catch (final InterruptedException ex) {
          //....
        } finally {
            service.shutdown();
        }
```
What do you think?

Reply:
@volodya-lombrozo Sure, I think we can.